lms/envs/common.py

```diff
@@ -2274,9 +2274,7 @@
         'base_url': os.path.join(MEDIA_URL, 'profile-images/'),
     },
 }
-PROFILE_IMAGE_DEFAULT_FILENAME = (
-    'images/edx-theme/default-profile' if FEATURES['IS_EDX_DOMAIN'] else 'images/default-theme/default-profile'
```

Comment:
@maxrothman Good point. I've changed it to only change the setting at all for edX domain.